### DISCUSSION TITLE
Remove the initial logo

### DISCUSSION THREAD

**Discussion by triquinielas** [ColorlibHQ/AdminLTE#5464] [Q&A]
When I run the web for the first time a banner with the logo appears, how can I remove it?
![header](https://github.com/ColorlibHQ/AdminLTE/assets/139678112/3e8d510c-82e4-478d-93cb-5e9edecf953b)
thanks in advance

**Comment by AstrooHiro** [comment]
![image](https://github.com/ColorlibHQ/AdminLTE/assets/113288171/4ebadb34-6a83-4be3-8d52-010c0ec5940a)

how did you access the index.html file?Apparently i can't...

**Reply by triquinielas** [reply]
Sorry, I don't understand you, I downloaded the code and ran it on my computer.
